docs/quickstart/bootstrap.rst

```diff
@@ -11,8 +11,8 @@ Mount this ISO file at the specific following path:
 
 .. code-block:: shell
 
-   root@bootstrap $ mkdir -p /srv/scality/metalk8s-|version|
-   root@bootstrap $ mount <path-to-iso> /srv/scality/metalk8s-|version|
+   root@bootstrap $ mkdir -p /srv/scality/metalk8s-2.0.0-dev
```

Comment:
Are we sure this is going to work? Last time I've tried changing this version number it failed.

Reply:
What do you mean by "it failed" ?